cabal.project

```diff
@@ -1,9 +1,14 @@
-packages:
+packages: 
          ./
          ghcide
 
 tests: true
-documentation: true
+
+program-options
+  ghc-options: -haddock
+
+package *
+  ghc-options: -haddock
```

Comment:
Why are two sets of `ghc-options` needed?

Reply:
I think the second one should suffice but the fact is `cabal configure --ghc-options=-haddock` generates both in the `cabal.project.local`. Otoh hls/ghcide is parsing the local modules with haddock enabled in hls so docs are shown even if you dont set the `ghc-option` at all (maybe you can correct me if that is wrong, but i *think* we were having docs in hover for local modules before any `-haddock` around)
So the first one could be removed, i only was being consistent with `cabal configure` output.

Reply:
Finally i've removed it, less is more